ui/src/components/SampleView/DrawGridPlugin.js

```diff
@@ -593,19 +593,31 @@ export default class DrawGridPlugin {
     }
   }
 
-  getClickedCell(shapeGroup, clickPoint) {
-    let cell = null;
+  getClickedCellIndex(gd, shapeGroup, e) {
+    const cellSizeX = this.getCellWidth(gd);
+    const cellSizeY = this.getCellHeight(gd);
 
-    shapeGroup.forEachObject((obj) => {
-      if (
-        obj.get('type') === 'ellipse' &&
-        obj.containsPoint(clickPoint, null, true)
-      ) {
-        cell = obj;
-      }
-    });
+    const cellIdxX = Number.parseInt(
+      Math.floor((e.offsetX - shapeGroup.oCoords.tl.x) / cellSizeX),
+      10,
+    );
+    const cellIdxY = Number.parseInt(
+      Math.floor((e.offsetY - shapeGroup.oCoords.tl.y - 20) / cellSizeY),
```

Comment:
Might be worth explaining why `- 20` in a comment, or making it a constant.

Reply:
:), yes. I honestly don't know why its 20 but I agree that a comment is useful